integrationTests/state/stateTrie/stateTrie_test.go

```diff
@@ -224,7 +224,7 @@ func TestAccountsDB_CommitTwoOkAccountsShouldWork(t *testing.T) {
 
 	rootHash, err := adb.RootHash()
 	assert.Nil(t, err)
-	fmt.Printf("Data committed! Root: %v\n", base64.StdEncoding.EncodeToString(rootHash))
+	fmt.Printf("data committed! Root: %v\n", base64.StdEncoding.EncodeToString(rootHash))
```

Comment:
Please revert here and on L315, L677, L693, L756, L772, L802, L823

Reply:
this is actually good

Reply:
compared to the other prints it looks awkward